config/opal_case_sensitive_fs_setup.m4

```diff
@@ -63,6 +63,8 @@ AC_ARG_WITH([cs_fs],
             [AS_HELP_STRING([--with-cs-fs],
                            [Destination FS is case sensitive (default: set to value of the build FS's case sensitivity)])])
 
+# Stupid emacs syntax hilighting: '
```

Comment:
Can you use dnl instead of # here?  Otherwise, the syntax highlighting of the output gets mucky when you're looking at the actual configure script.

Reply:
Will do.